src/commands/deployments/create.ts

```diff
@@ -1,27 +1,76 @@
-import { CommandModule } from "yargs";
+import { ArgumentsCamelCase, CommandModule } from "yargs";
 import { ERROR_MESSAGES } from "../../util/errors";
 import { getDeploymentApiClient } from "../../util/getClient";
-import { ResponseError } from "../../../sdk-client";
+import { Deployment, ResponseError } from "../../../sdk-client";
+
+export const createDeployment = async (args: ArgumentsCamelCase<{
```

Comment:
extracting this logic out of the handler anon function so that it can be re-used in `deploy.ts`